RecoLocalTracker/SubCollectionProducers/src/ClusterMultiplicityFilter.cc

```diff
@@ -15,19 +15,15 @@
 #include <memory>
 
 #include "RecoLocalTracker/SubCollectionProducers/interface/ClusterMultiplicityFilter.h"
-
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
-
-#include "DataFormats/SiStripCluster/interface/SiStripCluster.h"
-#include "DataFormats/Common/interface/DetSetVector.h"
-
 #include "Geometry/TrackerGeometryBuilder/interface/TrackerGeometry.h"
 
 
-ClusterMultiplicityFilter::ClusterMultiplicityFilter(const edm::ParameterSet& iConfig)
+ClusterMultiplicityFilter::ClusterMultiplicityFilter(const edm::ParameterSet& iConfig) :
+  maxNumberOfClusters_(iConfig.getUntrackedParameter<unsigned int>("MaxNumberOfClusters")),
+  clusterCollectionLabel_(iConfig.getUntrackedParameter<std::string>("ClusterCollectionLabel"))
```

Comment:
use InputTag directly in the configuration: conversions from string input parameter to InputTag should be avoinded